ansible/examples/slave_libvirt_static.yml

```diff
@@ -0,0 +1,240 @@
+---
+# This playbook is used to configure static libvirt&&vagrant slaves
+
+- hosts: all
+  become: true
+  user: ubuntu
+  vars:
+   - jenkins_user: 'jenkins-build'
+   # jenkins API credentials:
+   - api_user: 'ceph-jenkins'
+   - token: '{{ token }}'
+   - api_uri: 'https://jenkins.ceph.com'
+   - nodename: '{{ ansible_hostname }}'
+   - labels: '{{ labels }}'
+   - use_jnlp: true
+
+  tasks:
+
+    # vagrant doesn't have repositories, this chacra repo will be better to have
+    # around and can get updates as soon as a new vagrant version is published via
+    # chacractl
+    - name: add the vagrant repository
+      apt_repository:
+        repo: "deb [trusted=yes] https://chacra.ceph.com/r/vagrant/latest/HEAD/ubuntu/xenial/flavors/default/ xenial main"
+        state: present
+
+    # Run the equivalent of "apt-get update" as a separate step
+    - apt: update_cache=yes
+      when: ansible_pkg_mgr  == "apt"
+
+    - name: Install DEB requirements
+      apt: name={{ item }} state=present
```

Comment:
We're only supporting Xenial at the moment so we can leave it apt.  But you're right, I can remove a lot of `when:` statements and a few tasks because of that.